libs/ledger-live-common/src/families/vechain/js-transaction.ts

```diff
@@ -0,0 +1,135 @@
+import { Account, TransactionCommon } from "@ledgerhq/types-live";
+import BigNumber from "bignumber.js";
+import { $Shape } from "utility-types";
+import { DEFAULT_GAS_COEFFICIENT, HEX_PREFIX, MAINNET_CHAIN_TAG } from "./constants";
+import { Transaction } from "./types";
+import { Transaction as ThorTransaction } from "thor-devkit";
+import { calculateFee, estimateGas, generateNonce, getBlockRef } from "./utils/transaction-utils";
+import { VTHO_ADDRESS } from "./contracts/constants";
+import VIP180 from "./contracts/abis/VIP180";
+import { calculateTransactionInfo } from "./utils/calculateTransactionInfo";
+import { isValid } from "./utils/address-utils";
+
+/**
+ * Create an empty VET or VTHO transaction
+ *
+ * @returns {Transaction}
+ */
+export const createTransaction = (): Transaction => {
+  const chainTag = MAINNET_CHAIN_TAG;
+
+  return {
+    family: "vechain",
+    body: {
+      chainTag,
+      // placeholder, if "empty" returns error on send modal open
```

Comment:
I don't understand this comment

Reply:
Mirko wrote it, seems like that blockref has to be initialized because otherwise, it will throw an error